src/bigint.rs

```diff
@@ -809,100 +809,68 @@ impl_to_biguint!(u16,  FromPrimitive::from_u16);
 impl_to_biguint!(u32,  FromPrimitive::from_u32);
 impl_to_biguint!(u64,  FromPrimitive::from_u64);
 
-// Cribbed from core/fmt/num.rs
-#[derive(Copy, Clone)]
-pub struct RadixFmt {
```

Comment:
That is strange. I suppose it was made `pub` by accident. Its only use was in the previous implementation of `to_str_radix`. As you say, it can't be created outside the crate.